modules/web/src/app/dynamic/enterprise/quotas/component.ts

```diff
@@ -113,6 +113,21 @@ export class QuotasComponent implements OnInit {
     this._unsubscribe.complete();
   }
 
+  hasCpuOrDefaulted(element: QuotaDetails): boolean {
+    const quota = element?.quota;
+    return !!(quota?.cpu || quota.cpu === 0);
+  }
+
+  hasMemoryOrDefaulted(element: QuotaDetails): boolean {
+    const quota = element?.quota;
+    return !!(quota?.memory || quota.memory === 0);
+  }
+
+  hasStorageOrDefaulted(element: QuotaDetails): boolean {
+    const quota = element?.quota;
+    return !!(quota?.storage || quota.storage === 0);
+  }
```

Comment:
I think we can add these checks directly in the template because these only added in single place.
For example, for CPU it can be `element.quota && (element.quota.cpu || element.quota.cpu === 0)`
Wdyt?